src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
+                },
                 {
                     "term": "Signatur (Signierung eines Zertifikats)",
                     "anchor": "signature",
-                    "description": "Eine digitale Signatur ist das elektronische Äquivalent zu einer manuellen Unterschrift. Sie wird durch ein kryptographische Funktionen aus dem zu unterschreibenden Dokument erzeugt, ist unmittelbar mit einem elektronischen Dokument verbunden und hat folgende Zielstellungen: <ul><li>Nichtabstreitbarkeit, d.&nbsp;h. die unterschreibende Instanz kann nicht nachträglich abstreiten, das Dokument unterzeichnet zu haben</li><li>Echtheit des Dokuments, d.&nbsp;h. die Prüfung der Signatur macht auch sichtbar, wenn das Dokument nachträglich verändert wurde</li><li>Authentizität des Dokuments, d.&nbsp;h. es kann festgestellt werden, von wem das Dokument unterzeichnet wurde </li></ul> <br/><br/> An eine Signatur werden daher hohe Anforderungen gestellt: <br/><ul><li>Die Signatur muss technisch überprüfbar sein </li><li>Für den Prüfenden muss es einfach sein, die Signatur zu prüfen – gleichzeitig muss es für jeden anderen schwer sein, die Unterschrift zu fälschen</li><li>Die Signatur ist immer mit dem Dokument verbunden, passt nur zu diesem einen Dokument und darf nicht auf ein anderes Dokument übertragbar sein </li><li>Der Unterzeichnende darf nicht in der Lage sein, eine Signatur in seinem Namen abzustreiten</li></ul>"
+                    "description": "Eine digitale Signatur ist das elektronische Äquivalent zu einer manuellen Unterschrift. Sie wird durch eine kryptographische Funktion aus dem zu unterschreibenden Dokument erzeugt, ist unmittelbar mit einem elektronischen Dokument verbunden und hat folgende Zielstellungen: <ul><li>Nichtabstreitbarkeit, d.h. die unterschreibende Instanz kann nicht nachträglich abstreiten, das Dokument unterzeichnet zu haben</li><li>Echtheit des Dokuments, d.h. die Prüfung der Signatur macht auch sichtbar, wenn das Dokument nachträglich verändert wurde</li><li>Authentizität des Dokuments, d.h. es kann festgestellt werden, von wem das Dokument unterzeichnet wurde </li></ul> An eine Signatur werden daher hohe Anforderungen gestellt: <ul><li>Die Signatur muss technisch überprüfbar sein </li><li>Für den Prüfenden muss es einfach sein, die Signatur zu prüfen – gleichzeitig muss es für jeden anderen schwer sein, die Unterschrift zu fälschen</li><li>Die Signatur ist immer mit dem Dokument verbunden, passt nur zu diesem einen Dokument und darf nicht auf ein anderes Dokument übertragbar sein </li><li>Der Unterzeichnende darf nicht in der Lage sein, eine Signatur in seinem Namen abzustreiten</li></ul>"

```

Comment:
The `&nbsp;` in `d.&nbsp;h.` have been reverted. They should be put back in.

Reply:
Will be changed in the next commit, thanks!